tool/src/org/antlr/v4/codegen/BlankOutputModelFactory.java

```diff
@@ -29,7 +30,7 @@
 	public ParserFile parserFile(String fileName) { return null; }
 
 	@Override
-	public Parser parser(ParserFile file) { return null; }
```

Comment:
at first glance it looks strange that lexer() function below does not also need an ATN.

Reply:
Yes, because lexer works a bit differently and gets atnData from somewhere else. I'll take a look if it's possible to unify the parameter.

Reply:
It depends on this code: https://github.com/antlr/antlr4/blob/dev/tool/src/org/antlr/v4/codegen/OutputModelController.java#L140-L152 There is no call to `lexer` method from `OutputModelController` but call to constructor directly.